core/src/main/java/com/orientechnologies/orient/core/storage/impl/local/paginated/atomicoperations/OAtomicOperation.java

```diff
@@ -169,6 +168,14 @@ public void pinPage(OCacheEntry cacheEntry) throws IOException {
     pageChangesContainer.pinPage = true;
   }
 
+  /**
+   * Add External WalRecord to the atomic operation
+   * @param record
```

Comment:
The empty parameter, javadoc, could you remove it.

Reply:
what do you mean empty parameter?